moveit_demo_nodes/run_move_group/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ install(TARGETS run_move_group
   EXPORT export_${PROJECT_NAME}
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
-  RUNTIME DESTINATION bin
+  RUNTIME DESTINATION lib/${PROJECT_NAME}
   INCLUDES DESTINATION include
 )
 ament_export_targets(export_${PROJECT_NAME})
```

Comment:
Also remove this line.  There is no library to export so this isn't needed.

Reply:
@tylerjw is the line you mention

`LIBRARY DESTINATION lib`

Reply:
This line: `ament_export_targets(export_${PROJECT_NAME})`

Reply:
```suggestion
```

Reply:
oh, sure commiting now.